package.json

```diff
@@ -29,7 +29,9 @@
       "artifactName": "jitsi-meet.${ext}",
       "category": "public.app-category.video",
       "darkModeSupport": true,
-      "hardenedRuntime": false,
+      "hardenedRuntime": true,
```

Comment:
Does this also work in the non-signed case?

Reply:
It should not, as the entitlements also get used to ask for permission.

Reply:
https://developer.apple.com/documentation/security/hardened_runtime?preferredLanguage=occ